tensorflow_datasets/image/chexpert.py

```diff
@@ -0,0 +1,114 @@
+"""https://stanfordmlgroup.github.io/competitions/chexpert/"""
+
+import csv
+import os
+
+import tensorflow_datasets.public_api as tfds
+import tensorflow as tf
+
+
+_DESCRIPTION = '''\
+CheXpert is a large dataset of chest X-rays and competition for automated chest 
+x-ray interpretation, which features uncertainty labels and radiologist-labeled 
+reference standard evaluation sets. It consists of 224,316 chest radiographs 
+of 65,240 patients, where the chest radiographic examinations and the associated 
+radiology reports were retrospectively collected from Stanford Hospital. Each 
+report was labeled for the presence of 14 observations as positive, negative, 
+or uncertain. We decided on the 14 observations based on the prevalence in the 
+reports and clinical relevance.
+'''
+
+_CITATION = '''\
+@article{DBLP:journals/corr/abs-1901-07031,
+  author    = {Jeremy Irvin and Pranav Rajpurkar and Michael Ko and Yifan Yu and Silviana Ciurea{-}Ilcus and Chris Chute and Henrik Marklund and Behzad Haghgoo and Robyn L. Ball and Katie Shpanskaya and Jayne Seekins and David A. Mong and Safwan S. Halabi and Jesse K. Sandberg and Ricky Jones and David B. Larson and Curtis P. Langlotz and Bhavik N. Patel and Matthew P. Lungren and Andrew Y. Ng},
+  title     = {CheXpert: {A} Large Chest Radiograph Dataset with Uncertainty Labels and Expert Comparison},
+  journal   = {CoRR},
+  volume    = {abs/1901.07031},
+  year      = {2019},
+  url       = {http://arxiv.org/abs/1901.07031},
+  archivePrefix = {arXiv},
+  eprint    = {1901.07031},
+  timestamp = {Fri, 01 Feb 2019 13:39:59 +0100},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/abs-1901-07031},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+'''
+
+# Path to images and category labels in data dir
+_DATA_DIR =                   'CheXpert-v1.0-small'
```

Comment:
we tend not to rjustify like this.
also, to ensure this works on non-posix filesystems, use `os.path.join` instead of hardcoding `"/"`